nav2_urdf_setup/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.5)
+project(sam_bot_description)
```

Comment:
The package should be `sam_bot_description` rather than `nav2_urdf_setup` so that they match